controller/structures.xml

```diff
@@ -28,25 +28,25 @@
                     <action id="delivery-properties" name="Properties"  url="/taoDeliveryRdf/DeliveryMgmt/editDelivery" group="content" context="instance">
                         <icon id="icon-edit"/>
                     </action>
-                    <action id="delivery-class-new" name="New class" url="/taoDeliveryRdf/DeliveryMgmt/addSubClass" context="resource" group="tree" binding="subClass">
+                    <action id="delivery-class-new" name="New class" url="/taoDeliveryRdf/DeliveryMgmt/addSubClass" context="resource" group="tree" binding="subClass" weight="9">
```

Comment:
https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/872775905/Conventional+Commits
please use `feat: ` in the commit to create a proper version of extensions